website/content/docs/internals/telemetry/enable-telemetry.mdx

```diff
@@ -0,0 +1,109 @@
+---
+layout: docs
+page_title: Enable Vault telemetry
+description: >-
+  Step-by-step guide to enabling telemetry gathering with Vault
+---
+
+# Enable Vault telemetry gathering
+
+Collect telemetry data from your Vault installation.
+
+## Before you start
+
+- **You must have Vault 1.14 or later installed and running**.
+- **You must have access to your [Vault configuration](/vault/docs/configuration) file**.
+
+
+## Step 1: Choose an aggregation agent
+
+@include 'telemetry/supported-aggregation-agents.mdx'
+
+## Step 2: Enable at least one audit device
+
+To include audit-related metrics, you must enable auditing on at least one device
+with the `vault audit enable` command. For example, to enable auditing for the
+`file` device and save the logs to `/var/log/vault_audit.log`:
+
+```shell-session
+$ vault audit enable file file_path=/var/log/vault_audit.log
+```
+
+By default, Enterprise installations replicate audit devices to the secondary
+performance nodes in a cluster. To limit performance replication for an audit
+device, use the `local` flag to mark the device as local to the current node:
+
+```shell-session
+$ vault audit enable file -local file_path=/var/log/vault_audit.log
+```
+
+
+## Step 3: Configure telemetry collection
+
+To configure telemetry collection, update the telemetry stanza in your Vault
+configuration with your collection preferences and aggregation agent details.
+
+For example, the following `telemetry` stanza configures Vault with the standard
+telemetry defaults and connects it to a Statsite agent running on the default
+port within a company intranet at `mycompany.statsite`:
+
```

Comment:
Why do we give as our config example a stanza containing the standard defaults?  The defaults aren't necessarily "good" settings, often they're the way they are simply because we didn't want to break backwards compatibility.  Moreover, since they're the defaults, no one would ever use these settings, they'd simply omit the defaults to have a minimal config stanza, right?

Anyway, I'm not exactly objecting, just wondering why we're doing this instead of providing an example or two of typical realistic configs (maybe we do that elsewhere, like in tutorials?)

Reply:
It's helpful to have all the common, available parameters in an example block for a setup guide so folks know what a full telemetry stanza might look like. We do that for 2 main reasons: (1) it encourages readers to consider how they may want to customize the values right away instead of hoping they take the time to read the configuration document, and (2) it provides a clear template for them to copy/paste an change (which includes deleting things they want to leave as the default).

The stanza uses the defaults because I assumed they were reasonable values, if there are *more* reasonable values I'm happy to use those instead as a best practice.

Reply:
> The stanza uses the defaults because I assumed they were reasonable values, if there are more reasonable values I'm happy to use those instead as a best practice.

Eh, they're not *unreasonable* for statsd.